### ISSUE TITLE
Adding a contribution section

### ISSUE THREAD

**Issue by no-mood** [cybersecurity-polito/.github#1]
Given the many questions, I think it would be a good idea to add a `Contributions` section. It should contain information about how to contribute via GitHub (fork the repository, clone your fork, push, open a PR). Additionally, since there are many LaTeX documents, a Quickstart section would be helpful because some people feel intimidated (or are simply lazy).

I'm not sure where to place the Contributions section, or whether to include the LaTeX quickstart within it or keep them separate.

Moreover, I think it would be beneficial to set up a nix flake or devenv in coding projects (such as LaTeX) so that people can easily set up the same environment. This promotes good practices and saves me the trouble of repeatedly explaining how to install LaTeX on their WSL.

**Comment by eferollo** [comment]
I agree with you on creating a contribution guide, particularly for LaTeX indentation and best practices that help produce well-organized, easy-to-read PDF notes. I'd suggest placing a `CONTRIBUTING.md` directly within the note template, along with an `INSTALL.md` to guide users on setting up the LaTeX environment and linking to basic resources for getting started with LaTeX.

On the other hand, I wouldn’t spend time explaining how to contribute on GitHub. We are engineering students here, one should figure out how to do that on their own...
And please, let's avoid useless Windows overhead. Anyone using Windows should handle any related issues alone.
